client/my-sites/stats/stats-chart-tabs/index.jsx

```diff
@@ -15,6 +15,8 @@ var ElementChart = require( 'components/chart' ),
 	observe = require( 'lib/mixins/data-observe' ),
 	StatsModulePlaceholder = require( '../stats-module/placeholder' ),
 	Card = require( 'components/card' );
+import UpgradeNudge from 'my-sites/upgrade-nudge';
```

Comment:
Because that creates merge conflict with any other PR editing anything within imports in this file. In my experience, once PR converts imports, it sits there while another author converts or adds imports in another PR.
Then rebasing is a pain and all if the code needs to be re-tested, because require can handle default export in a different way than es6.
I will gladly convert in a different PR.